setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```

Comment:
Btw I am not sure this is needed as `black` formats lines to max 88 (or smth like that).

Reply:
Good point! However, when not specified, flake has the upper limit of 79, so it would still scream

Reply:
Ah, funny :). But let's put it to 88 so we are inline with black? It is indeed 88 https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html

Reply:
Right, thanks for the source, I will change it.

They even say `If you’re using Flake8, you can bump max-line-length to 88 and mostly forget about it.`

Reply:
Great :+1: